changelog/updates/2024-08-23-azure-nvme-utils-update.md

```diff
@@ -0,0 +1 @@
+- azure-nmve-utils ([0.2.0](https://github.com/Azure/azure-nvme-utils/releases/tag/v0.2.0))
```

Comment:
Nit: `s/nmve/nvme/`

I have just manually fixed in the release notes draft.

Reply:
Woops, thanks. Ironically, the project just renamed itself to azure-vm-utils!